torchgeo/datasets/utils.py

```diff
@@ -216,6 +218,22 @@ def download_radiant_mlhub_collection(
     collection.download(output_dir=download_root, api_key=api_key)
 
 
+def collate_patches_per_tile(batch: List[Dict[str, Any]]) -> Dict[str, Any]:
```

Comment:
Can we move this below and name it to match our other collation functions (stack/merge/unbind samples)? Should also export it in `__init__.py` like we do for the other ones.